### PR TITLE
[NLU 1383]

### PR DESCRIPTION
**Pull request by samhickey25**
Fork snips-nlu-parsers to allow for resolution of dates before 1970

### CODE REVIEW

File: python/ci.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+          """
+          withCredentials([usernamePassword(credentialsId: '14645c89-70a0-4b46-a8a2-2f0a580d13dd', usernameVariable: 'user', passwordVariable: 'password')]) {
+            sh """
+            rm -rf pip.conf
+            echo '[global]' >> pip.conf
+            echo 'extra-index-url = https://$user:$password@purecloud.jfrog.io/purecloud/api/pypi/inin-pypi/simple' >> pip.conf
+            """
+          }
+          createLocalStorageFolders()
+          createTestReportDir()
```
Comment (schutza) [inline python/ci.jenkinsfile]: these are not needed, you can remove them

File: python/ci.jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+}
+def createLocalStorageFolders() {
+  LOCAL_STORAGE = "data"
+  sh """
+  rm -Rf \$(pwd)/${LOCAL_STORAGE}
+  mkdir -p \$(pwd)/${LOCAL_STORAGE}/dynamodblocal
+  mkdir -p \$(pwd)/${LOCAL_STORAGE}/minio
+  chmod -R 777 \$(pwd)/${LOCAL_STORAGE}
+  """
+}
```
Comment (schutza) [inline python/ci.jenkinsfile]: remove those

File: python/snips_nlu_parsers/__version__ [hunk 3]
```diff
@@ -1 +1 @@
-0.4.3
+0.1.1
```
Comment (schutza) [inline python/snips_nlu_parsers/__version__]: no newline at EOF

File: python/setup.py [hunk 4]
```diff
@@ -9,9 +9,9 @@
 
 packages = [p for p in find_packages() if "tests" not in p]
 
-PACKAGE_NAME = "snips_nlu_parsers"
+PACKAGE_NAME = "snips_nlu_parsers_genesys"
 ROOT_PATH = os.path.dirname(os.path.abspath(__file__))
-PACKAGE_PATH = os.path.join(ROOT_PATH, PACKAGE_NAME)
+PACKAGE_PATH = os.path.join(ROOT_PATH, "snips_nlu_parsers")
```
Comment (schutza) [inline python/setup.py:14]: I presume that package path is intended?

**Review by schutza — CHANGES_REQUESTED** [review]
Overall LGTM, just clean up the ci.jenkinsfile and that should be good

File: python/setup.py (hunk 4 above)
Comment (samhickey25) [reply python/setup.py:14]: Yes, the path is pointing to the python/snips_nlu_parsers directory. Package name is changed to reflect that we are forking the snips package

**Review by schutza — APPROVED** [review]

**Review by ushmau5 — APPROVED** [review]

**Review by sanhal — APPROVED** [review]

**Review by colm-dillon — APPROVED** [review]

**Pull request merged**